lighthouse-core/report/html/renderer/pwa-category-renderer.js

```diff
@@ -54,7 +54,7 @@ class PwaCategoryRenderer extends CategoryRenderer {
   renderScoreGauge(category, groupDefinitions) {
     // Defer to parent-gauge style if category error.
     if (category.score === null) {
-      return super.renderScoreGauge(category, groupDefinitions);
```

Comment:
I think we want false here or we need a fix somewhere else?

![image](https://user-images.githubusercontent.com/2301202/57275004-ec433480-7051-11e9-97a5-8008ba6a2b6c.png)

Reply:
oops yea, there are two gauge templates and I forgot to handle the pwa one.